ydb/core/blobstorage/vdisk/common/vdisk_mongroups.h

```diff
@@ -517,6 +517,8 @@ public:
 
                 COUNTER_INIT_IF_EXTENDED(PutTotalBytes, true);
                 COUNTER_INIT_IF_EXTENDED(GetTotalBytes, true);
+                
+                COUNTER_INIT(MinHugeBlobInBytes, false);
```

Comment:
Counter of this type is useless. We need a histogram-like counter, with one label indicating minimum huge blob size, and the sensor value of 1 for specific vdisk.